#!/usr/bin/env python
PACKAGE = "aa_signs"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("x_angle_min", int_t, 0, "min x angle", 70, 0, 180);
gen.add("x_angle_max", int_t, 0, "max x angle", 110, 0, 180);
gen.add("y_angle_min", int_t, 0, "min y angle", 80, 0, 180);
gen.add("y_angle_max", int_t, 0, "max y angle", 100, 0, 180);
gen.add("z_angle_min", int_t, 0, "min z angle", 0, 0, 180);
gen.add("z_angle_max", int_t, 0, "max z angle", 180, 0, 180);


gen.add("x_min", int_t, 0, "min x", -10000, -10000, 10000);
gen.add("x_max", int_t, 0, "max x", 10000, -10000, 10000);
gen.add("y_min", int_t, 0, "min y", -10000, -10000, 10000);
gen.add("y_max", int_t, 0, "max y", 10000, -10000, 10000);
gen.add("z_min", int_t, 0, "min z", -10000, -10000, 10000);
gen.add("z_max", int_t, 0, "max z", 10000, -10000, 10000);


gen.add("blur_depth", int_t, 0, "size of bluring depth image", 5, 1, 100);
gen.add("blur_angles", int_t, 0, "size of bluring angle image", 5, 1, 100);

exit(gen.generate(PACKAGE, "aa_signs", "signDetection"))